lib/cadet/assessments/assessments.ex

```diff
@@ -3,29 +3,132 @@ defmodule Cadet.Assessments do
   Assessments context contains domain logic for assessments management such as
   missions, sidequests, paths, etc.
   """
-  use Cadet, :context
+  use Cadet, [:context, :display]
 
-  alias Cadet.Assessments.QuestionTypes.MCQQuestion
-  alias Cadet.Assessments.QuestionTypes.ProgrammingQuestion
+  import Ecto.Query
 
-  # To be uncommented when assessments context is merged
+  alias Timex.Duration
+
+  alias Cadet.Assessments.Assessment
+  alias Cadet.Assessments.Question
+
+  def all_assessments() do
+    Repo.all(Assessment)
+  end
+
+  def all_assessments(category) do
+    Repo.all(from(a in Assessment, where: a.category == ^category))
+  end
+
+  def all_open_assessments(category) do
+    now = Timex.now()
+
+    assessment_with_category = Repo.all(from(a in Assessment, where: a.category == ^category))
```

Comment:
Can use the `all_assessments(category)` method above but I think just leave it for now. In general I think we should combine this and the `Enum.filter` below into one query string so that the DB engine only gives us the results we want instead of fetching too much data into memory and then filtering it. Maybe leave that for a later refactor.

Reply:
Good point. I'll write it down as a TODO